src/Subject.ts

```diff
@@ -5,7 +5,7 @@ import { Subscriber } from './Subscriber';
 import { ISubscription, Subscription, TeardownLogic } from './Subscription';
 import { ObjectUnsubscribedError } from './util/ObjectUnsubscribedError';
 import { SubjectSubscription } from './SubjectSubscription';
-import { $$rxSubscriber } from './symbol/rxSubscriber';
+import { rxSubscriber as Symbol_rxSubscriber } from './symbol/rxSubscriber';
```

Comment:
haha... GOTCHA! `rxSubscriber` doesn't exist on `Symbol`, really... (i.e. `Symbol.rxSubscriber` is not at thing)... so you can probably just leave this as `rxSubscriber` or `as rxSubscriberSymbol` or something)